components/tests.py

```diff
@@ -176,16 +173,18 @@ def setUpTestData(cls):
         Component.objects.create(
             title="Another Even Cooler Component",
             description="This one is better.",
-            catalog=Catalog.objects.get(id=cls.test_catalog.id),
+            catalog=Catalog.objects.get(id=test_catalog.id),
             controls=TEST_COMPONENT_CONTROLS,
             search_terms=["cool", "best"],
             type="software",
             component_json=TEST_COMPONENT_JSON_BLOB,
         )
 
+        cls.test_catalog = test_catalog
```

Comment:
Didn't mean to add this in this PR. I'll remove.

Reply:
Actually, I realized that that class attribute was never used, so I'm going to go ahead and update.